streams-tests/jvm/src/test/scala/zio/stream/SinkSpec.scala

```diff
@@ -244,6 +244,24 @@ object SinkSpec extends ZIOBaseSpec {
           assertM(sinkIteration(sink, "123").either, isLeft(equalTo("Ouch")))
         }
       ),
+      suite("drop")(
+        testM("happy path") {
+          val sink = ZSink.identity[Int].drop(1)
+          assertM(sinkIteration(sink, 1).either, isLeft(equalTo(())))
+        },
+        testM("init error") {
+          val sink = initErrorSink.drop(1)
+          assertM(sinkIteration(sink, 1).either, isLeft(equalTo("Ouch")))
+        },
+        testM("step error") {
+          val sink = stepErrorSink.drop(1)
+          assertM(sinkIteration(sink, 1).either, isLeft(equalTo("Ouch")))
+        },
+        testM("extract error") {
+          val sink = extractErrorSink.drop(1)
```

Comment:
Can you please add tests that drop more than one element? As also, one that drops zero element. 🙏